.github/workflows/test.yml

```diff
@@ -0,0 +1,14 @@
+name: Test
+
+on:
+  push:
+
+jobs:
+  tests:
+    name: Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Unit tests
+        run: docker-compose run --rm php ./run-tests.sh
```

Comment:
Not seeing the need for docker-compose. Does `./run-tests.sh` work?

Reply:
It might work with a tweak or two, but I think we would like to standardize on running all CI processes in a Docker environment that we design and control.